File: CS/PerformanceCounterCorrelationExample/TwoCounterSample.cs
namespace PerformanceCounterCorrelation {
    internal class TwoCounterSample {
        public float CounterOne { get; set; }
        public float CounterTwo { get; set; }

        public TwoCounterSample(float x, float y) {
            CounterOne = x;
            CounterTwo = y;
        }
    }
}

File: VB/PerformanceCounterCorrelationExample/TwoCounterSample.vb
Namespace PerformanceCounterCorrelation

    Friend Class TwoCounterSample

        Public Property CounterOne As Single

        Public Property CounterTwo As Single

        Public Sub New(ByVal x As Single, ByVal y As Single)
            CounterOne = x
            CounterTwo = y
        End Sub
    End Class
End Namespace
